core/src/main/java/org/elasticsearch/search/aggregations/bucket/MultiBucketsAggregation.java

```diff
@@ -64,6 +66,12 @@
 
         Object getProperty(String containingAggName, List<String> path);
 
+        @Override
+        default void readFrom(StreamInput in) throws IOException {
+            // NORELEASE remove me when no Buckets override it
+            throw new UnsupportedOperationException("Prefer the Writeable interface");
+        }
```

Comment:
Ah, wait, was reading it wrong. We need this because I'm slowly removing `Streamable`. The buckets don't extend `InternalAggregation` so they don't get this stub.